docs/content/preview/releases/release-notes/v2.12.md

```diff
@@ -17,6 +17,68 @@ Included here are the release notes for all releases in the v2.12 release series
 
 For an RSS feed of the release notes for all release series, point your feed reader to the [RSS feed for releases](../../index.xml).
 
+## v2.12.12.0 - February 10, 2023 {#v2.12.12.0}
+
+**Build:** `2.12.12.0-b12`
+
+**Third-party licenses:** [YugabyteDB](https://downloads.yugabyte.com/releases/2.12.12.0/yugabytedb-2.12.12.0-b12-third-party-licenses.html), [YugabyteDB Anywhere](https://downloads.yugabyte.com/releases/2.12.12.0/yugabytedb-anywhere-2.12.12.0-b12-third-party-licenses.html)
+
+### Downloads
+
+<ul class="nav yb-pills">
```

Comment:
It's worth doing a `curl -I <download-url>` for all three download packages and both license files before you publish, to make sure they actually exist.